semantic/lookup.go

```diff
@@ -1,26 +1,26 @@
 package semantic
 
 import (
+	"fmt"
 	flatbuffers "github.com/google/flatbuffers/go"
 	"github.com/influxdata/flux/codes"
 	"github.com/influxdata/flux/internal/errors"
 	"github.com/influxdata/flux/libflux/go/libflux"
 	"github.com/influxdata/flux/semantic/internal/fbsemantic"
 )
 
+var byteArr = libflux.EnvStdlib()
+var typeEnv = fbsemantic.GetRootAsTypeEnvironment(byteArr, 0)
+var environment = TypeEnvMap(typeEnv)
+
 // LookupBuiltInType returns the type of the builtin value for a given
-// Flux stdlib package. Returns a empty MonoType if lookup fails.
+// Flux stdlib package. Returns an error if lookup fails.
 func LookupBuiltInType(pkg, name string) (MonoType, error) {
-	byteArr := libflux.EnvStdlib()
-	env := fbsemantic.GetRootAsTypeEnvironment(byteArr, 0)
 	var table flatbuffers.Table
-	// perform a lookup of package identifier in flatbuffers TypeEnvironment
-	prop, err := lookup(env, pkg, name)
-	if err != nil {
-		return MonoType{}, err
-	}
+	prop := environment[pkg][name] // query environment map for prop
+
 	if !prop.V(&table) {
-		return MonoType{}, errors.Newf(codes.Internal, "Prop value is not valid: %v", err)
+		return MonoType{}, errors.New(codes.Internal, "Prop value is not valid")
```

Comment:
There isn't really a pertinent error to add here, unless anyone has a suggestion.

Reply:
You have a package path and the name, it would be good to put that in the error that you return.